CONTRIBUTING.md

```diff
@@ -4,7 +4,7 @@ We'd love to accept your patches and contributions to this project. There are ju
 
 ## Code of conduct
 
-Please comply with the [code of conduct](https://documentation.gxf.lfenergy.org/Opensourcecommunity/Code-of-conduct.html) 
+Please comply with the [code of conduct](https://grid-exchange-fabric.gitbook.io/gxf/opensourcecommunity/code-of-conduct) 
```

Comment:
Volgens mij kun je de links relatief maken, dat lost alle issues op.

Reply:
The suggestion to use relative links looks very useful, though I doubt relative links will solve **all** issues anywhere.

Please keep in mind though that comments on pull requests in the GXF repositories are supposed to be in English, not in Dutch.